bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/types/BResourceMethodType.java

```diff
@@ -53,9 +53,9 @@ public String toString() {
         }
         StringJoiner sj = new StringJoiner(",", "resource function " + accessor + " " + rp.toString() +
                 "(", ") returns (" + type.retType + ")");
-        for (int i = 0; i < parameters.length; i++) {
-            Type type = parameters[i].type;
-            sj.add(type.getName() + " " + parameters[i].name);
+        for (var parameter : parameters) {
```

Comment:
Shall we avoid using `var` (to help with readability).

Reply:
Fixed in [1663d93](https://github.com/ballerina-platform/ballerina-lang/pull/42905/commits/1663d93c8796e217939ab8765f7911a6f4372592)